### ISSUE TITLE
Radio Buttons are not focusable with Button Type Styles Layout

### ISSUE THREAD

**Issue by SethFalco** [fluentform/fluentform#386]
## Steps to Reproduce

1. Create a form with a **Radio Field**
2. Under **Advanced Options** → **Layout**, select **Button Type Styles**
3. Preview the form

For accessibility, it should be possible to focus on the form elements with keyboard navigation or assistive technologies.

Currently, these elements can only be interacted with by mouse control or touch displays.

## Proposal

* The radio buttons (or equivalent `span`) should have `tabindex="0"` to indicate to the browser that they should be focusable elements.
* They should have styles applied on focus (`:focus`) so users can see that it's focused.
